docs/source/package_reference/cli.md

```diff
@@ -228,6 +228,26 @@ The following arguments are only useful when training in SageMaker
 * `--aws_access_key_id AWS_ACCESS_KEY_ID` (`str`) -- The AWS_ACCESS_KEY_ID used to launch the Amazon SageMaker training job
 * `--aws_secret_access_key AWS_SECRET_ACCESS_KEY` (`str`) -- The AWS_SECRET_ACCESS_KEY used to launch the Amazon SageMaker training job
 
+## accelerate estimate-memory
+
+**Command**:
+
+`accelerate estimate-memory` or `accelerate-estimate-memory` or `python -m accelerate.commands.estimate`
+
+Estimates the total vRAM a particular model hosted on the Hub needs to perform Big Model Inference and training.
+
+**Usage**: 
+
+```bash
+accelerate estimate-memory [arguments]
+```
+
+**Optional Arguments**:
+
+* `--model_name MODEL_NAME` (`str`)-- The model name on the Hugging Face Hub
```

Comment:
This is not optional at all, not passing this will result in an error. I think it's more natural to do `accelerate estimate-memory model_name` then have to type the `--model_name` on top of that.

Reply:
Fixed